etc/cloud_nightly.yml

```diff
@@ -1 +1,48 @@
-# Evergreen configuration for MongoDB Cloud E2E tests with MongoDB
+stepback: true
+
+# Needed til EVG-3714 is deployed
+functions:
+  something:
+  - command: shell.exec
+    params:
+      shell: bash
+      script: |
+        exit
+
+tasks:
+- name: GENERATE_MMS_TESTS
+  commands:
+  - command: shell.exec
+    params:
+      shell: bash
+      script: |
+        set -ex
+        git clone --branch ${cloud_branch} --depth 1 git@github.com:${cloud_repo} mms
+        cd mms/scripts
+        ./init_virtualenv.sh ${workdir} 'no-om'
+        VIRTUAL_ENV_DISABLE_PROMPT=true source venv-activate
+        cd evergreen
+        sleep ${compile_wait_duration_seconds}
+        ./mongo_cloud_nightly/generate_cloud_nightly.py > ${workdir}/cloud_nightly.json
```

Comment:
[question] Is the Evergreen task meant to fail if the compile task hadn't run? I wasn't sure what definition for "red", "green", "purple", and "lavender" the Cloud team had wanted to use and what return code the `generate_cloud_nightly.py` script would exit with if the compile task hadn't run.

Reply:
This task should always succeed because it will always generate tasks just based on manipulating fragments of YAML we already have in the cloud repo. It would only fail if someone made incompatible changes to the YAML, or if there were a bug in the script that generates task definitions.

In general, the generated tasks will not fail if the compile fails since there's nothing we can do about that on our end. I have a separate status code for this in the script that waits for tasks to finish.